coffee/expression.py

```diff
@@ -39,7 +39,7 @@ class MetaExpr(object):
 
     """Metadata container for a compute-intensive expression."""
 
-    def __init__(self, type, parent, loops_info, domain):
+    def __init__(self, type, parent, loops_info, domain, mode=0):
```

Comment:
Docstring says "suggested rewrite mode". Can you elaborate? Isn't it the optimisation level?